lib/src/target/target_focus.dart

```diff
@@ -18,6 +18,7 @@ class TargetFocus {
     this.alignSkip,
     this.paddingFocus,
     this.focusAnimationDuration,
+    this.unFocusAnimationDuration,
```

Comment:
Its already in the newest version 1.2.2

Reply:
In 1.2.2, the `unFocusAnimationDuration` is only in `TutorialCoachMark`. I add it to the `TargetFocus`, so that the `unFocusAnimationDuration` can be adjust in each `TargetFocus`.